src/main/java/ama/WebSecurityConfiguration.java

```diff
@@ -31,7 +31,6 @@ public BCryptPasswordEncoder bCryptPasswordEncoder() {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
-                .csrf().disable()
```

Comment:
We had to disable since we couldnt pass csrf tokens in our previous tests. Now that the tests use Mockmvc we can also pass the token within the request. So we dont need to disable.